workspace/cmd/api2/api2.go

```diff
@@ -0,0 +1,71 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"log"
+	"net/http"
+
+	apiv2 "github.com/bufbuild/buf-examples/workspace/gen/proto/go/api/v2"
+	"github.com/bufbuild/buf-examples/workspace/gen/proto/go/api/v2/apiv2connect"
+	"github.com/bufbuild/connect-go"
+	"golang.org/x/net/http2"
+	"golang.org/x/net/http2/h2c"
+	emptypb "google.golang.org/protobuf/types/known/emptypb"
+)
+
+var ErrorNeedsImplementation = errors.New("needs implementation")
+
+func main() {
+	const addr = "127.0.0.1:8080"
+
+	log.Printf("Observability service starting")
+
+	mux := http.NewServeMux()
+
+	path, handler := apiv2connect.NewObservabilityServiceHandler(&service{})
+	mux.Handle(path, handler)
```

Comment:
nit: No real reason to use a new mux here, you could just use the default mux with `http.Handle` and pass `http.DefaultServeMux` in as the `h2c.NewHandler`.

Reply:
Yeah, you are right. Removed.